langchain/callbacks/manager.py

```diff
@@ -133,14 +180,40 @@ def trace_as_chain_group(
     *,
     project_name: Optional[str] = None,
     example_id: Optional[Union[str, UUID]] = None,
+    inheritable_tags: Optional[List[str]] = None,
```

Comment:
I think we should offer only inheritable (which means "will be propagated to child runs") tags here, and call them `tags` only

Reply:
Yeah that works